weather_sp/setup.py

```diff
@@ -26,7 +26,7 @@
     packages=find_packages(),
     author='Anthromets',
     author_email='anthromets-ecmwf@google.com',
-    version='0.1.0',
+    version='0.2.0',
```

Comment:
What determines that this is 0.2.0 while the other one is now 0.1.1? From the Versions & Release text it sounds like all tools should have the same version. Can you clarify this a bit more in the explanation? Is it ok if major / minor versions of the tools drift apart?

Reply:
Yeah, that's a great question. I'm following the [semver.org](http://semver.org)'s rules of thumb. I'm updating the minor version since `weather-sp` gained a backwards compatible new feature, namely the output path structuring things you recently landed. If that were considered a bugfix, then I would increment the version to `0.1.1`. 

I could probably provide an example in the release process notes – Each specific tool will have it's own version, and we'll increment the versions of the whole `weather-tools` package if any subtool is updated.